rsocket-core/src/main/java/io/rsocket/core/RSocketConnector.java

```diff
@@ -395,6 +399,23 @@ public RSocketConnector lease(Supplier<Leases<? extends LeaseStats>> supplier) {
     return this;
   }
 
+  /**
+   * When this is set, frames reassembler control maximum payload size which can be reassembled.
+   *
+   * <p>By default this is not set in which case maximum reassembled payloads size is not
+   * controlled.
+   *
+   * @param maxPayloadSize the threshold size for reassembly, must no be less than 64 bytes
+   * @return the same instance for method chaining
+   * @see <a
+   *     href="https://github.com/rsocket/rsocket/blob/master/Protocol.md#fragmentation-and-reassembly">Fragmentation
+   *     and Reassembly</a>
+   */
+  public RSocketConnector maxPayloadSize(int maxPayloadSize) {
```

Comment:
Still doubt whether that is the best name. 

What about `maxReceivedPayloadSize` since this is applicable for received payloads, but not for the sent. WDYT @rstoyanchev?

Reply:
Also, I would add linking to Transport#maxFrameSize saying that maxReceivedPayloadSize must be >= to maxFrameSize, otherwise received frame can exit the maxReceivedPayloadSize

Reply:
Good point. Perhaps `maxInboundPayloadSize` and `maxInboundFrameSize`?